packages/js/product-editor/src/components/header/header.tsx

```diff
@@ -77,6 +78,10 @@ export function Header( {
 		[ productType, productId ]
 	);
 
+	const [ visibleTab, setVisibleTab ] = useState< string | null >(
```

Comment:
The Header is responsible to change the `selectedTab` so I think we should use that instead of having a new state here.
You just need to add it to the Headers props like follow
```
<Header
  selectedTab={ selectedTab }
  ...
/>
```

Reply:
Good call! Done in [aea1db7](https://github.com/woocommerce/woocommerce/pull/48192/commits/aea1db77acf1a684a87646408ba03df811ef53a3)